scene/main/viewport.cpp

```diff
@@ -688,6 +688,18 @@ void Viewport::_process_picking() {
 		physics_picking_events.clear();
 		return;
 	}
+#ifndef _3D_DISABLED
+	if (use_xr) {
+		if (XRServer::get_singleton() != nullptr) {
+			Ref<XRInterface> xr_interface = XRServer::get_singleton()->get_primary_interface();
+			if (xr_interface.is_valid() && xr_interface->is_initialized() && xr_interface->get_view_count() > 1) {
+				WARN_PRINT_ONCE("Object picking can't be used when stereo rendering, this will be turned off!");
+				physics_object_picking = false; // don't try again.
+				return;
+			}
+		}
+	}
+#endif
```

Comment:
Could we move this entire branch into `set_physics_object_picking()` so that we don't have it inside the process function?

Reply:
I'll have a look at that when I have a moment.

Reply:
So the problem here is checking `get_view_count` which we can't do while setting the properties as the interface will not be setup until a couple of frames in.

One of these days I want to redo how we handle viewports and move the logic into the Viewport node instead of hidden inside of the rendering engine. Not today however.